docs/reference/reference-deploy/config/agent-config-reference.rst

```diff
@@ -106,3 +102,5 @@
       "shutdown", "now"]``, or just ``["shutdown", "now"]`` if the agent is running as root.
       Additional system configuration may be required in order to allow the agent to execute the
       command from inside a Docker container or without the need to enter a password.
+
+-  ``label``: Deprecated field. Use ``resource_pool`` instead.
```

Comment:
Deprecated to me somewhat implies it might still work, maybe removed to be more explicit?

Reply:
I vote for removing deprecated/ignored fields from references entirely.